How-To/Connect-to-AWS-Resources.md

```diff
@@ -105,35 +105,35 @@ As with R, enabling [unload](https://laughingman7743.github.io/PyAthena/pandas.h
     # Load necessary libraries
     import os
     import pandas
+    import pyarrow
     from pyathena import connect
     from pyathena.pandas.util import as_pandas
     from pyathena.pandas.cursor import PandasCursor
 
     # Connect to Athena
-    conn = connect(
+    cursor = connect(
         # We add '+ "/"' to the end of the line below because enabling unload
-        # introduces a concatenation error in the pyathena package not present
-        # when the option is disabled.
+        # requires that the staging directory end with a slash
         s3_staging_dir=os.getenv("AWS_ATHENA_S3_STAGING_DIR") + "/",
         region_name=os.getenv("AWS_REGION"),
         cursor_class=PandasCursor,
     ).cursor(unload=True)
 
-    # Define test query
-    SQL_QUERY = "SELECT * from default.vw_pin_sale LIMIT 10;"
+    # Define test query. Note that the query CANNOT end with a semi-colon
+    SQL_QUERY = "SELECT * from default.vw_pin_sale LIMIT 10"
 
     # Execute query and return as pandas df
     cursor.execute(SQL_QUERY)
 
-    df = as_pandas(cursor)
+    df = cursor.as_pandas()
```

Comment:
We're already using the `PandasCursor` class, so using the built-in method actually makes this slightly faster.

Reply:
This is great news.